fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/AstBuilder.java

```diff
@@ -396,6 +398,48 @@ protected ParseNode visitValues(Values node, ParseTreeContext context) {
         }
     }
 
+    @Override
+    protected ParseNode visitUnnest(Unnest node, ParseTreeContext context) {
+        List<Expr> arguments = visit(node.getExpressions(), context, Expr.class);
+        List<Expr> expressions = new ArrayList<>();
+
+        for (Expr arg : arguments) {
+            if (arg instanceof ArrayExpr) {
+                if (!arg.getChildren().isEmpty()) {
+                    // need to covert array[row(1,2), row(3,4))] to array[1,3], array[2,4], so SR could unnest it
+                    Expr firstArrayElement = arg.getChildren().get(0);
```

Comment:
We only support `array[row(literals)]` this format?

Reply:
or array[literals]